ZEP000-prevent-ood.md

```diff
@@ -0,0 +1,137 @@
+---
+title: ZEP Enable brokers to recover from out of disk space due to exporter failure
+authors:
+  - Deepthi Akkoorath
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2020-07-02
+last-updated: 2020-07-02
+status: implemented
+
+---
+
+# Summary
+[summary]: #summary
+
+When exporter is slow or not available, it prevents the broker from taking snapshot and compacting the log. Eventually the broker goes out of disk space. This ZEP proposes a solution to prevent the broker from reaching a non-recoverable state. When the exporter is healthy again, the broker should be able to compact the log eventually.
+
+The broker's strategy to prevent out of disk space is as follows. When the broker's disk space usage goes above a threshold
```

Comment:
```suggestion
The broker's strategy to prevent out of disk space is to react when the broker's disk space usage goes above a threshold, it will:
```